tests/cases/resource_test.py

```diff
@@ -455,10 +455,26 @@ def testGetResourceByPath(self):
                 str(resp.json['_id']), str(item['_id']))
 
         # test bogus path
+        # test is not set
         resp = self.request(path='/resource/lookup',
                             method='GET', user=self.user,
-                            params={'path': '/bogus/path'})
+                            params={'path': '/collection/bogus/path'})
```

Comment:
the '/bogus/path' isn't really checking for a path that doesnt exists though, it just instantly fails as it is not in the proper path format. So maybe have both? One that checks for a bogus path, and one that checks for improper path formatting?